app/Listeners/VisitTransfer/NotifyRefereeOfReferenceCancellation.php

```diff
@@ -15,7 +15,6 @@ public function __construct()
 
     public function handle(ReferenceCancelled $event)
     {
-        $reference = $event->reference;
-        $reference->application->account->notify(new ApplicationReferenceCancelled($reference));
+        $event->reference->notify(new ApplicationReferenceNoLongerNeeded($event->reference));
```

Comment:
We should ensure that this event only fires to relevant referees (using the static method REFERENCE_IS_PENDING) to avoid sending users unnecessary notifications

Reply:
This already happens elsewhere, see https://github.com/VATSIM-UK/core/pull/1171/files#diff-fda47f2e5dc0e1b3b11e181ac8d7a0fdR279